FishBunDemo/src/main/java/com/sangcomz/fishbundemo/ImageAdapter.kt

```diff
@@ -0,0 +1,48 @@
+package com.sangcomz.fishbundemo
+
+import android.net.Uri
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.ImageView
+import androidx.recyclerview.widget.RecyclerView
+import com.squareup.picasso.Picasso
+import java.util.*
+
+/**
+ * Created by sangc on 2015-11-06.
+ */
+class ImageAdapter(
+    private var imageController: ImageController,
+    private var imagePaths: ArrayList<Uri>
```

Comment:
Fixed it. Btw, I didn't know about emptyList(). When is it mainly used? Could you explain to me?

8c0fd08

Reply:
`val` property must be initialized. So instead of the code below, It is used that.
```kotlin
val list:List<..> = new ArrayList<>()
```
Otherwise, this property must be declared `var` type.